std/math.d

```diff
@@ -328,6 +335,9 @@ template floatTraits(T)
             enum EXPPOS_SHORT = 0;
             enum SIGNPOS_BYTE = 0;
         }
+
+        import std.typecons : Tuple;
+        alias Integral = Tuple!(ulong, "significand", ushort);
```

Comment:
Except that simply reversing the fields' order won't work because of alignment
issues, so I'll have to create an ad hoc version of Tuple here, or extend the
original one to allow specifying alignment. This is doable, but it is definitely
more work than I'm ready to devote to supporting a format for which I have no
simple and straightforward way of testing and which was not supported before,
especially since this PR is not supposed to add new functionality, being merely
a rewrite of already existing code.